thorlcr/graph/thgraphslave.cpp

```diff
@@ -1433,6 +1435,16 @@ class CThorCodeContextSlave : public CThorCodeContextBase, implements IEngineCon
         // NB. includes access to foreign Dalis.
         return jobChannel.queryJob().getOptBool("slaveDaliClient");
     }
+    virtual StringBuffer &getQueryId(StringBuffer &result, bool isShared) const
+    {
+        return result.append(jobChannel.queryJob().queryWuid());
```

Comment:
MORE: Possibility this could be shared between graphs on the same job, but if so it would be hard to clean up quickly.